.git-blame-ignore-revs

```diff
@@ -1,3 +1,6 @@
+# Autofix eslint curly rule.
+0221522f253e094b277a1485b7a2d186cb172632
+
```

Comment:
I'm not sure if we want to maintain chronological order for revs, but that is what I usually did for this file.